docs/installation/server-config/security.rst

```diff
@@ -132,28 +133,213 @@ Configuring Authorization of KSQL with Kafka ACLs
 Kafka clusters can use ACLs to control access to resources. Such clusters require each client to authenticate as a particular user.
 To work with such clusters, KSQL must be configured to :ref:`authenticate with the Kafka cluster <config-security-ssl-sasl>`,
 and certain ACLs must be defined in the Kafka cluster to allow the user KSQL is authenticating as access to resources.
-The list of ACLs that must be defined depends on whether the KSQL cluster is configured for
-:ref:`interactive <config-security-ksql-acl-interactive>` or :ref:`non-interactive (headless) <config-security-ksql-acl-headless>`.
+The list of ACLs that must be defined depends on the version of the Kafka cluster.
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+|cp| v5.0 (Apache Kafka v2.0) and above
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+|cp| 5.0 simplifies the ACLs required to run KSQL against a Kafka cluster secured with ACLs,
+(See `KIP-277 <https://cwiki.apache.org/confluence/display/KAFKA/KIP-277+-+Fine+Grained+ACL+for+CreateTopics+API>`__ and
```

Comment:
Grammar nit: "ACLs, (See ...)." -> "ACLs (see ...)."

Reply:
I've converted See -> see, but left the comma - cos that's proper english, init.